dali/operators/generic/slice/subscript.h

```diff
@@ -239,12 +238,17 @@ class TensorSubscript : public Operator<Backend> {
                                 " while processing sample #", i));
         }
 
-        if (lo < 0)
-          lo += in_extent;
-        if (hi < 0)
-          hi += in_extent;
-        lo = clamp(lo, 0_i64, in_extent - 1);
-        hi = clamp(hi, 0_i64, in_extent);
+        if (in_extent > 0) {
```

Comment:
Without this, the line `lo = clamp(lo, 0_i64, in_extent - 1)` gets swapped lo/hi bounds and produces nonsensical results.